extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -448,51 +509,56 @@ class QueueProcessor {
                 log.error('an error occurred when putting metadata to S3',
                           { method: 'QueueProcessor._putMetadata',
                             entry: entry.getLogInfo(),
-                            origin: this.destConfig.s3,
+                            origin: this.destBackbeatHost,
                             error: err.message });
                 return cbOnce(err);
             }
             return cbOnce(null, data);
         });
     }
 
-    _setupClients(sourceRole, targetRole, log) {
-        const sourceS3 = this.sourceConfig.s3.host;
-        // FIXME use bootstrap list
-        const [destHost, destPort] = this.destConfig.bootstrapList[0].servers
-            .split(':');
-
+    _setupSourceClients(sourceRole, log) {
         this.s3sourceAuthManager =
-            this._createAuthManager(this.sourceConfig.auth, sourceRole, log);
-        this.s3destAuthManager =
-            this._createAuthManager(this.destConfig.auth, targetRole, log);
+            this._createAuthManager('source', this.sourceConfig.auth,
+                                    sourceRole, log);
 
         // Disable retries, use our own retry policy (mandatory for
         // putData route in order to fetch data again from source).
 
+        const sourceS3 = this.sourceConfig.s3;
         this.S3source = new AWS.S3({
-            endpoint: `${this.sourceConfig.transport}://` +
+            endpoint: `${sourceS3.transport}://` +
```

Comment:
You are right, we made these config changes in an earlier PR, the changes may have been reverted during conflict resolution.
@jonathan-gramain Can you rever the changes for transport config